api-inference-community/docker_images/spacy/app/pipelines/token_classification.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import sys
+from typing import Any, Dict, List
+
+from app.pipelines import Pipeline
+
+
+class TokenClassificationPipeline(Pipeline):
+    def __init__(
+        self,
+        model_id: str,
+    ):
+        # At the time, only public models from spaCy are allowed in the inference API.
+        full_model_path = model_id.split("/")
+        if len(full_model_path) != 2:
+            return
```

Comment:
Shouldn't those be exceptions ?

Reply:
Should the exception instead be in `__call__` so we can propagate the error to the response in `pipeline_route`? I don't think that `startup_event` has any error handling at the moment.

Reply:
We need to check for sure, I believe failing to load a pipeline should be ignored, and subsequent calls should retry loading so failure should happen in __init__. 

(I think it makes code much easier to write to leave exceptions were they should be, we should probably account for that in the wrapper)